src/packages/shared-types/opensearch/main/transforms/seatool.ts

```diff
@@ -130,23 +147,28 @@ export const transform = (id: string) => {
         (item) => item.SPW_STATUS_ID === data.STATE_PLAN.SPW_STATUS_ID
       )?.SPW_STATUS_DESC || "Unknown";
     const { stateStatus, cmsStatus } = getStatus(seatoolStatus);
+    const authorityId = data.PLAN_TYPES?.[0].PLAN_TYPE_ID;
+    const typeId = data.STATE_PLAN_SERVICETYPES?.[0]?.SERVICE_TYPE_ID;
+    const subTypeId = data.STATE_PLAN_SERVICE_SUBTYPES?.[0]?.SERVICE_SUBTYPE_ID;
     return {
       id,
+      flavor: flavorLookup(data.STATE_PLAN.PLAN_TYPE), // This is MEDICAID CHIP or WAIVER... our concept
```

Comment:
Three values are listed in the comment here, but the `Flavor` type also includes "SPA". What case would that fourth value be present here?